src/node/blockstorage.h

```diff
@@ -143,6 +138,19 @@ class BlockManager
     std::unordered_map<std::string, PruneLockInfo> m_prune_locks GUARDED_BY(::cs_main);
 
 public:
+    using Options = kernel::BlockManagerOpts;
+
+    explicit BlockManager(const Options& opts)
+        : m_prune_mode{opts.prune_mode},
+          m_prune_target{opts.prune_target} {};
+
+    std::atomic<bool> m_importing{false};
+    /** Pruning-related variables and constants */
+    /** True if we're running in -prune mode. */
+    const bool m_prune_mode;
+    /** Number of bytes of block files that we're trying to stay below. */
+    const uint64_t m_prune_target;
```

Comment:
I would prefer if these were private and only accessed through methods on `BlockManager`.

Reply:
done, I think